hnn_core/tests/test_network.py

```diff
@@ -17,11 +17,11 @@ def test_network():
     params_fname = op.join(hnn_core_root, 'param', 'default.json')
     params = read_params(params_fname)
     # add rhythmic inputs (i.e., a type of common input)
-    params.update({'input_dist_A_weight_L2Pyr_ampa': 5.4e-5,
-                   'input_dist_A_weight_L5Pyr_ampa': 5.4e-5,
+    params.update({'input_dist_A_weight_L2Pyr_ampa': 1.4e-5,
+                   'input_dist_A_weight_L5Pyr_ampa': 2.4e-5,
                    't0_input_dist': 50,
-                   'input_prox_A_weight_L2Pyr_ampa': 5.4e-5,
-                   'input_prox_A_weight_L5Pyr_ampa': 5.4e-5,
+                   'input_prox_A_weight_L2Pyr_ampa': 3.4e-5,
+                   'input_prox_A_weight_L5Pyr_ampa': 4.4e-5,
```

Comment:
why were these changed @ntolley ? or did I mess up somewhere with the rebase?

Reply:
This just reflects that the bug can result in all cells taking the same weights when they share a dictionary.

I think the reason we didn't notice this bug in the first place is because we often used the same weights for all cells when adding a bursty input.